states/OK/upper.yaml

```diff
@@ -0,0 +1,58 @@
+contact_form:
+  steps:
+    - visit: "http://www.oksenate.gov/Senators/biographies/SendEmails.aspx"
+    - fill_in:
+        - name: TextBoxName
+          selector: "#TextBoxName"
+          value: $NAME_FULL
+          required: true
+        - name: TextBoxAddress
+          selector: "#TextBoxAddress"
+          value: $ADDRESS_STREET
+          required: true
+        - name: TextBoxCity
+          selector: "#TextBoxCity"
+          value: $ADDRESS_CITY
+          required: true
+        - name: TextBoxState
+          selector: "#TextBoxState"
+          value: $ADDRESS_STATE_POSTAL_ABBREV
+          required: true
+        - name: TextBoxZip
+          selector: "#TextBoxZip"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: TextBoxPhoneNumber
+          selector: "#TextBoxPhoneNumber"
+          value: $PHONE
```

Comment:
Should we include the phone number, since it's not required by the form?  I know we discussed and I think we landed at omitting if not required....

Reply:
I've included phone number where possible on all the other state legislature forms -- since there isn't the communication-becomes-public-record concern I'd prefer to include it.

Reply:
Fair enough!